pypd/models/integration.py

```diff
@@ -76,5 +76,5 @@ def create(cls, service=None, endpoint=None, data=None, *args, **kwargs):
             endpoint = 'services/{0}/integrations'.format(sid)
 
         # otherwise endpoint should contain the service path too
-        getattr(Entity, 'create').__func__(cls, endpoint=endpoint, data=data,
-                                          *args, **kwargs)
+        return getattr(Entity, 'create').__func__(cls, endpoint=endpoint,
```

Comment:
According to the [Python documentation for this method](https://docs.python.org/2/library/functions.html#getattr), `getattr` should already return the value. Can you clarify the issue you're running into that prompted adding this `return` statement?

Reply:
Yes, `getattr` will retrieve the value of `Entity`'s `create` attribute, but it will not surface it to the caller of `create`. As is, `create` returns `None` like any default Python function.

A super simplistic example:
```
$ cat example.py
froop = "example"


def func(thing):
    getattr(thing, "__len__")


print(func(froop))
```
Gives us:
```
$ python example.py
None
```
Whereas:
```
$ cat example.py
froop = "example"


def func(thing):
    return getattr(thing, "__len__")


print(func(froop))
```
Gives us:
```
$ python example.py
<method-wrapper '__len__' of str object at 0x100a766c0>
```

It's not a great example, but it's easier/shorter than pasting here the entire module I have written around this.

Furthermore, my suggestion is consistent with every single other method in the library, please see:
* [`return`](https://github.com/PagerDuty/pagerduty-api-python-client/blob/master/pypd/models/incident.py#L44)
* [`return`](https://github.com/PagerDuty/pagerduty-api-python-client/blob/master/pypd/models/incident.py#L101)
* [`return`](https://github.com/PagerDuty/pagerduty-api-python-client/blob/master/pypd/models/incident.py#L123)
* [`return`](https://github.com/PagerDuty/pagerduty-api-python-client/blob/master/pypd/models/incident.py#L145)

To name just a few.

Even in the same file, `integration.py`, [`fetch`](https://github.com/PagerDuty/pagerduty-api-python-client/blob/master/pypd/models/integration.py#L48) implements this correctly. The expectation here is the same: create the resource, and return the object of the newly created resource. The only difference between the two functions is that `fetch` merely retrieves an assumed existing integration, and `create` asks to create it first.

Thank you for your attention! ❤️